### ISSUE TITLE
OrganizationActivity SubEvent

### ISSUE THREAD

**Issue by aduchon** [cloudbees/zendesk-java-client#24]
Getting this error when pulling down some data:

 org.zendesk.client.v2.ZendeskException: java.lang.IllegalArgumentException: Could not resolve type id 'OrganizationActivity' into a subtype of [simple type, class org.zendesk.client.v2.model.events.Event]
 at [Source: N/A; line: -1, column: -1](through reference chain: org.zendesk.client.v2.model.Audit["events"])

**Comment by johnou** [comment]
@aduchon would you kindly provide some code to reproduce this exception?

**Comment by matthewtckr** [comment]
Reproduction example, based upon [Zendesk docs](https://developer.zendesk.com/rest_api/docs/core/ticket_audits#organization-subscription-notification-event):

```java
    String json = "{ \"id\": 21337631753, \"type\": \"OrganizationActivity\", \"subject\": null, \"body\": \"This is sample data\", \"recipients\": [568628833] }";
    ObjectMapper mapper = new ObjectMapper();
    try {
    Event ev = mapper.readValue( json.getBytes(), Event.class );
    } catch (Exception e ) {
      System.out.println( e );
    }
```

**Comment by matthewtckr** [comment]
This can be closed as fixed in 0.3.1.  Thanks.
